api-module-library/microsoft-teams/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@friggframework/api-module-microsoft-teams",
-    "version": "0.4.0",
+    "version": "0.4.1",
```

Comment:
@brendanrmoore we actually don't need to bump the version! Although we have done plenty of that lately. It should default to consider this a patch update, but we can also label the PR.

Reply:
@MichaelRyanWebber ahh, I see. I reverted that commit anyway to clean up the PR. If it looks good, do you mind merging it in? Thanks for reviewing!